src/modules/form/components/DynamicField.tsx

```diff
@@ -413,7 +414,11 @@ const DynamicField: React.FC<DynamicFieldProps> = ({
         return <MultiNameInput {...objProps} />;
       }
       if (item.component == Component.Address) {
-        return <MultiAddressInput {...objProps} />;
+        return item.repeats ? (
+          <MultiAddressInput {...objProps} />
+        ) : (
+          <SingleAddressInput {...objProps} />
+        );
       }
       if (item.component == Component.Phone) {
```

Comment:
Once a move in date is collected, it should display on the page - I hacked together the below by adding the snippet to `DynamicViewField`

<img width="775" alt="Screenshot 2023-11-01 at 9 59 51 AM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/3795f513-9a2b-49c7-9d0b-9cfa0ed6d718">


```
    case ItemType.Object:
      if (item.component == Component.Address && value) {
        return item.repeats ? (
          <>todo stack addresses</>
        ) : (
          <ClientAddress address={Array.isArray(value) ? value[0] : value} />
        );
      }
```